src/languages/es.js

```diff
@@ -250,18 +250,19 @@ export default {
         },
     },
     signInPage: {
-        expensifyDotCash: 'Expensify.cash',
-        expensifyIsOpenSource: 'Expensify.cash es open source',
+        expensifyDotCash: 'New Expensify',
+        expensifyIsOpenSource: 'New Expensify es open source',
         theCode: 'el código',
         openJobs: 'trabajos disponibles',
     },
     termsOfUse: {
-        phrase1: 'Al usar Expensify.cash, estás aceptando los',
+        phrase1: 'Al usar Nuevo Expensify, estás aceptando los',
```

Comment:
The english version of this leaves out the name of the app. I think we should do the same here.

English:
> By logging in, you agree to the

@iwiznia @marklouisdeshaun what is the proper translation into Spanish?

Reply:
> By logging in, you agree to the

Al iniciar sesión, estás accediendo a los

Reply:
Thanks!

Reply:
Does this read correctly, then?

![image](https://user-images.githubusercontent.com/1228807/126190321-47782626-9003-4322-9092-f300fcc4c5c3.png)

Reply:
Nope, seems it's missing a line? Is [this](https://github.com/Expensify/Expensify.cash/blob/tgolen-password-copy/src/languages/en.js#L274-L281) the up to date copy?
```
    termsOfUse: {
        phrase1: 'By logging in, you agree to the',
        phrase2: 'terms of service',
        phrase3: 'and',
        phrase4: 'privacy policy',
        phrase5: 'Money transmission is provided by Expensify Payments LLC (NMLS',
        phrase6: 'ID:2017010) pursuant to its',
        phrase7: 'licenses',
    },
```
If so:
```
    termsOfUse: {
        phrase1: 'Al usar Expensify.cash, estás aceptando los',
        phrase2: 'términos de servicio',
        phrase3: 'y',
        phrase4: 'política de privacidad',
        phrase5: 'El envío de dinero es brindado por Expensify Payments LLC (NMLS',
        phrase6: 'ID:2017010) de conformidad con sus',
        phrase7: 'licencias',
    },
```

Reply:
Oh, so sorry... I was looking at the welcome text 🤦 

OK, I think I've got it right if you want to check the changes in this PR.